packages/bruno-app/src/components/Sidebar/Collections/Collection/index.js

```
@@ -38,6 +38,7 @@ import { isTabForItemActive } from 'src/selectors/tab';
 import RenameCollection from './RenameCollection';
 import StyledWrapper from './StyledWrapper';
 import CloneCollection from './CloneCollection';
+import { areItemsLoading } from 'utils/collections';
```

Biome (2.4.4)

[error] Shouldn't redeclare 'areItemsLoading'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)